packages/docs/src/pages/components/upload/examples/ExRounded.vue

```diff
@@ -22,18 +22,20 @@
                     {{ file2.name }}
                 </span>
             </b-upload>
-        </b-field> 
+        </b-field>
     </b-field>
   </section>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
```

Comment:
The [same comment](https://github.com/ntohq/buefy-next/pull/269/files#r1732964052) is applicable.